backend/backend/core/ollama.py

```diff
@@ -1,33 +1,53 @@
 import httpx
 
-from langchain_ollama import ChatOllama
+from pydantic.json_schema import JsonSchemaValue
+from typing import Literal, Union
 
-from utils.logger import logger
+from langchain_ollama import ChatOllama
 
 from config import settings
+from utils.logger import logger
 
 
 class OllamaBackend(object):
     def __init__(self):
         self._pull_models()
 
     def _pull_models(self):
-        logger.info(f"Pulling Ollama model '{settings.OLLAMA_MODEL}' at endpoint '{
-                    settings.OLLAMA_ENDPOINT}' ...")
+        logger.info(
+            f"Pulling Ollama model '{settings.OLLAMA_MODEL}' at endpoint '{
+                    settings.OLLAMA_ENDPOINT}' ..."
+        )
         with httpx.Client() as client:
             response = client.post(
                 f"{settings.OLLAMA_ENDPOINT}/api/pull",
-                json={"name": settings.OLLAMA_MODEL})
+                json={"name": settings.OLLAMA_MODEL},
+            )
             for chunk in response.iter_lines():
```

Comment:
The `response.iter_lines()` method is not necessary here, as we are only making a POST request. Consider removing this line to simplify the code.